### DISCUSSION TITLE
My Profile screen clarity suggestions

### DISCUSSION THREAD

**Discussion by maze88** [streetcomplete/StreetComplete#5537] [Feedback]
The _My Profile_ screen has some really cute data, however its context is unclear - is ranking between all OSM users or just StreetComplete users? if the later - how many actually are they?

Suggestions:
- Change the _My Profile_ sreen's title to _StreetComplete Profile_ (if indeed that's the case).
- Display how many users one is ranked "against" - for example if I am rank 3 in my country - I'd like to know if that's out of 5 users or 500 users! Suggestions:
  - > 3 Out of 16 in \<my country>
  - > 123 Out of all 999 StreetComplete Users
  
  If publishing the total number of users is privacy-violating or intrusive, perhaps an order of magnitude can be shown, for example "... Out of ~100...".

**Comment by westnordost** [comment]
Why would this information be worth to include?

**Reply by maze88** [reply]
Having the [existing] rank data is fun, interesting and motivating (looking at quests/goals like a game)... however, what I'm trying to say, is that from a glance (at least IMHO) it's not entirely clear if it's in the scope of OSM or only StreetComplete.
